apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -1319,6 +1320,49 @@ export class CaseService {
     })
   }
 
+  getConnectedIndictmentCases(
+    caseId: string,
+    defendant: Defendant,
+  ): Promise<Case[]> {
+    let whereClause: WhereOptions = {
+      type: CaseType.INDICTMENT,
+      id: { [Op.ne]: caseId },
+      state: [CaseState.RECEIVED],
+    }
```

Comment:
I have to initialize it and reassign because I'm adding one of 2 possible extra options to it depending on how the national id is formatted (nationalId vs birth date)